docs/receiving/receiving-with-ingest.mdx

```diff
@@ -0,0 +1,150 @@
+---
+title: Receiving Webhooks with Ingest
+---
+
+import CodeTabs from '@theme/CodeTabs';
+import TabItem from '@theme/TabItem';
+
+Receiving webhooks with [Svix Ingest] starts with creating a `Source`.
+
+A `Source` generates an endpoint you can share with a webhook provider as a
+destination for their webhooks.
+
+Ingest supports a variety of signature verification schemes and flows used by
+webhook providers. Supported values for a `Source`'s' `type` include:
+
+- `beehiiv`
+- `brex`
+- `clerk`
+- `github`
+- `guesty`
+- `hubspot`
+- `incidentIo`
+- `lithic`
+- `nash`
+- `pleo`
+- `replicate`
+- `resend`
+- `safebase`
+- `sardine`
+- `shopify`
+- `stripe`
+- `stych`
+- `svix` (that's us!)
+- `zoom`
+
+Additionally there's the option to configure a `Source` as `http` to skip
+performing signature verification. This is useful for providers that have no
+verification scheme and for providers whose verification scheme is not yet
+supported by Ingest.
+
+:::tip
+Let us know if your webhook provider isn't yet supported so we can add it!
+:::
+
+
+## Create a Source
+
+Getting started with Ingest is a matter of configuring n `Source`.
+
+This can be done from either the API:
+
+```shell
+curl https://api.svix.com/ingest/api/v1/source \
+  -H "Authorization: Bearer $SVIX_TOKEN" \
+  -H "Content-Type: application/json" \
+  -d @- << EOF
+{
+  "name": "demo",
+  "uid": "demo",
+  "type": "github",
+  "config": {
+    "secret": "<the signing secret>"
+  }
+}
+EOF
+```
+
+or from the Ingest Dashboard:
+
+![screenshot of the Ingest Dashboard showing the overview tab for a Source named "demo"](../../static/img/ingest/source-edit.png)
+
+
+## Tell your provider where to send webhooks
+
+The `IngestSourceOut` response from the API will include an `ingestUrl` which is
+the endpoint you give to your provider, telling them where to send their
+webhooks.
+
+The `ingestUrl` is also listed on the Ingest Dashboard for each `Source`.
+
+For GitHub, as is used in this example, this is referred to as the
+_"Payload URL"_.
+
+
+## Managing incoming messages
+
+In the Ingest Dashboard, the Destinations tab for your Source is where you can
```

Comment:
link to Ingest dashboard?

Reply:
I linkified any place it's mentioned like this.